recipe/meta.yaml

```diff
@@ -14,7 +14,7 @@ source:
 
 build:
   number: 1
-  skip: true  # [ppc64le or (cuda_compiler_version != "11.2")]
+  skip: true  # [ppc64le or (cuda_compiler_version not in ["11.2", "12.0"])]
```

Comment:
Seems like this need some kind of tweak

Reply:
This change looks fine to me, any concern?

Reply:
Maybe we want this instead?

```suggestion
  skip: true  # [ppc64le or not (cuda_compiler_version in ["11.2", "12.0"])]
```